templates/components/workflows/python/deep_research/README-template.md

```diff
@@ -49,6 +49,18 @@ curl --location 'localhost:8000/api/chat' \
 --data '{ "messages": [{ "role": "user", "content": "Create a report comparing the finances of Apple and Tesla" }] }'
 ```
 
+## Customize the UI
+
+To customize the UI, you can start by modifying the [./components/deep_research_event.jsx](./components/deep_research_event.jsx) file.
+
+You can also generate a new UI component for the workflow by running the following command:
+
+```
+poetry run generate:ui --input_file ./app/workflow.py --output_file ./components/deep_research_event.jsx
```

Comment:
what happens if workflow has more than one UIEvent? as ./components/deep_research_event.jsx can be only one file?

seems to me that if you have multiple UIEvents, you need to specify the output_dir:
```
poetry run generate:ui --input_file ./app/workflow.py --output_dir ./components
```

but as 
```
poetry run generate
``` 
has no parameters, why does `generate:ui` needs some?

you can just have it like this:
```suggestion
poetry run generate:ui 
```
and then hardcode the files in the code